CMakePresets.json

```diff
@@ -0,0 +1,121 @@
+{
+  "version": 6,
+  "configurePresets": [
+    {
+      "name": "_root-config",
+      "hidden": true,
+      "generator": "Ninja",
```

Comment:
Should this be `Ninja Multi-Config` as updated to [Dockerfile](https://github.com/beman-project/exemplar/pull/39/files#diff-74d629d3c2ebfd97a4acab7c9c2cdfcae721b98e92873131bac63b6454cdad46L29-R29) in #39 ?

@camio @ComicSansMS

Reply:
Yes, exactly. Feel free to do that in a separate PR though because this file will have to be refactored since there will be only one `gcc` preset needed after that.

Reply:
I am not familiar with that, I will need to look into it. let's ship this PR then!